src/query/storage/m3/storage.go

```diff
@@ -257,20 +260,28 @@ func (s *m3storage) Write(
 	id := query.Tags.ID()
```

Comment:
Maybe we should consider renaming WriteQuery to WriteRequest or something, query implies a response to a question

Reply:
used at a lot of places. I'd prefer to avoid that change in this diff